cmd/snap-gpio-helper/cmd_export.go

```diff
@@ -33,5 +35,43 @@ type cmdExportChardev struct {
 }
 
 func (c *cmdExportChardev) Execute(args []string) error {
-	return errors.New("not implemented")
+	chipLabels := strings.Split(c.Args.ChipLabels, ",")
+	filter := func(chip GPIOChardev) bool {
+		for _, label := range chipLabels {
+			if chip.Label() == label {
+				return true
+			}
+		}
+		return false
+	}
+	chips, err := findChips(filter)
+	if err != nil {
+		return err
+	}
+	if len(chips) == 0 {
+		return errors.New("no matching gpio chips found matching passed labels")
+	}
+	if len(chips) > 1 {
+		return errors.New("more than one gpio chips were found matching passed labels")
+	}
+
+	chip := chips[0]
+	if err := validateLines(chip, c.Args.Lines); err != nil {
+		return fmt.Errorf("invalid lines argument: %w", err)
+	}
```

Comment:
this can prob be done earlier

Reply:
I need the matched chip to cross check the referenced lines with the chip lines